### DISCUSSION TITLE
html file created with pydeck polygon layer - how to get it online?

### DISCUSSION THREAD

**Discussion by RicardoGomesIST** [visgl/deck.gl#7537] [Q&A]
Hi,
I am using pydeck - polygon layer.
Everything run fine but at the end I have a html file with 80Mb
Is there a way to reduce it size? I would like to present my map online but I when I added to my github repository it does not work. I just added the html file that works in my desktop. Is it from the file size or I also need some libraries together with it?
thank you

**Comment by kylebarron** [comment]
> html file with 80Mb

If you use pydeck to save an HTML file, it saves your entire dataset embedded in the HTML file. So an 80MB HTML file is mostly your data

**Reply by kylebarron** [reply]
whoops on a flight with slow internet and didn't mean to make three threads, and can't delete the other two.
